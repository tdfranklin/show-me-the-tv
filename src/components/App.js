import React, { Component } from 'react';
import darkBaseTheme from 'material-ui/styles/baseThemes/darkBaseTheme';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import Navbar from './navbar/Navbar';
import Search from './search/Search';

class App extends Component {

    render() {
        return (
            <MuiThemeProvider muiTheme={getMuiTheme(darkBaseTheme)}>
                <div>
                    <Navbar />
                    <Search />
                </div>                
            </MuiThemeProvider>
        );
    }
}

export default App;