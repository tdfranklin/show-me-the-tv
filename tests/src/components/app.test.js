import React from 'react';
import ReactDOM from 'react-dom';
import App from '../../../src/components/App';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('App', () => {
    it('matches the snapshot', () => {
        const tree = shallow(<App />);
        expect(toJson(tree)).toMatchSnapshot();
    });
});