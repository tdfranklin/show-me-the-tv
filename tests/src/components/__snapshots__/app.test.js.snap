// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the snapshot 1`] = `
<MuiThemeProvider
  muiTheme={
    Object {
      "appBar": Object {
        "color": "#0097a7",
        "height": 64,
        "padding": 24,
        "textColor": "#303030",
        "titleFontWeight": 400,
      },
      "avatar": Object {
        "backgroundColor": "rgb(101, 101, 101)",
        "color": "#303030",
      },
      "badge": Object {
        "color": "#303030",
        "fontWeight": 500,
        "primaryColor": "#0097a7",
        "primaryTextColor": "#303030",
        "secondaryColor": "#ff4081",
        "secondaryTextColor": "#303030",
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "baseTheme": Object {
        "fontFamily": "Roboto, sans-serif",
        "palette": Object {
          "accent1Color": "#ff4081",
          "accent2Color": "#f50057",
          "accent3Color": "#ff80ab",
          "alternateTextColor": "#303030",
          "borderColor": "rgba(255, 255, 255, 0.3)",
          "canvasColor": "#303030",
          "clockCircleColor": "rgba(255, 255, 255, 0.12)",
          "disabledColor": "rgba(255, 255, 255, 0.3)",
          "pickerHeaderColor": "rgba(255, 255, 255, 0.12)",
          "primary1Color": "#0097a7",
          "primary2Color": "#0097a7",
          "primary3Color": "#757575",
          "secondaryTextColor": "rgba(255, 255, 255, 0.7)",
          "shadowColor": "rgba(0, 0, 0, 1)",
          "textColor": "rgba(255, 255, 255, 1)",
        },
        "spacing": Object {
          "desktopDrawerMenuItemHeight": 48,
          "desktopDropDownMenuFontSize": 15,
          "desktopDropDownMenuItemHeight": 32,
          "desktopGutter": 24,
          "desktopGutterLess": 16,
          "desktopGutterMini": 8,
          "desktopGutterMore": 32,
          "desktopKeylineIncrement": 64,
          "desktopSubheaderHeight": 48,
          "desktopToolbarHeight": 56,
          "iconSize": 24,
        },
      },
      "borderRadius": 2,
      "bottomNavigation": Object {
        "backgroundColor": "#303030",
        "height": 56,
        "selectedColor": "#0097a7",
        "selectedFontSize": 14,
        "unselectedColor": "rgba(255, 255, 255, 0.54)",
        "unselectedFontSize": 12,
      },
      "button": Object {
        "height": 36,
        "iconButtonSize": 48,
        "minWidth": 88,
      },
      "card": Object {
        "fontWeight": 500,
        "subtitleColor": "rgba(255, 255, 255, 0.54)",
        "titleColor": "rgba(255, 255, 255, 0.87)",
      },
      "cardMedia": Object {
        "color": "rgba(255, 255, 255, 0.87)",
        "overlayContentBackground": "rgba(0, 0, 0, 0.54)",
        "subtitleColor": "rgba(255, 255, 255, 0.54)",
        "titleColor": "rgba(255, 255, 255, 0.87)",
      },
      "cardText": Object {
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "checkbox": Object {
        "boxColor": "rgba(255, 255, 255, 1)",
        "checkedColor": "#0097a7",
        "disabledColor": "rgba(255, 255, 255, 0.3)",
        "labelColor": "rgba(255, 255, 255, 1)",
        "labelDisabledColor": "rgba(255, 255, 255, 0.3)",
        "requiredColor": "#0097a7",
      },
      "chip": Object {
        "backgroundColor": "rgb(72, 72, 72)",
        "deleteIconColor": "rgba(255, 255, 255, 0.26)",
        "fontSize": 14,
        "fontWeight": 400,
        "shadow": "0 1px 6px rgba(0, 0, 0, 0.12),
        0 1px 4px rgba(0, 0, 0, 0.12)",
        "textColor": "rgba(255, 255, 255, 0.87)",
      },
      "datePicker": Object {
        "calendarTextColor": "rgba(255, 255, 255, 1)",
        "calendarYearBackgroundColor": "#303030",
        "color": "#0097a7",
        "headerColor": "rgba(255, 255, 255, 0.12)",
        "selectColor": "#0097a7",
        "selectTextColor": "#303030",
        "textColor": "#303030",
      },
      "dialog": Object {
        "bodyColor": "rgba(255, 255, 255, 0.6)",
        "bodyFontSize": 16,
        "titleFontSize": 22,
      },
      "drawer": Object {
        "color": "#303030",
        "width": 256,
      },
      "dropDownMenu": Object {
        "accentColor": "rgba(255, 255, 255, 0.3)",
      },
      "enhancedButton": Object {
        "tapHighlightColor": "rgba(0, 0, 0, 0)",
      },
      "flatButton": Object {
        "buttonFilterColor": "#999999",
        "color": "rgba(0, 0, 0, 0)",
        "disabledTextColor": "rgba(255, 255, 255, 0.3)",
        "fontSize": 14,
        "fontWeight": 500,
        "primaryTextColor": "#0097a7",
        "secondaryTextColor": "#ff4081",
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "floatingActionButton": Object {
        "buttonSize": 56,
        "color": "#0097a7",
        "disabledColor": "rgb(72, 72, 72)",
        "disabledTextColor": "rgba(255, 255, 255, 0.3)",
        "iconColor": "#303030",
        "miniSize": 40,
        "secondaryColor": "#ff4081",
        "secondaryIconColor": "#303030",
      },
      "fontFamily": "Roboto, sans-serif",
      "gridTile": Object {
        "textColor": "#ffffff",
      },
      "icon": Object {
        "backgroundColor": "#0097a7",
        "color": "#303030",
      },
      "inkBar": Object {
        "backgroundColor": "#ff4081",
      },
      "isRtl": false,
      "listItem": Object {
        "leftIconColor": "#757575",
        "nestedLevelDepth": 18,
        "rightIconColor": "#757575",
        "secondaryTextColor": "rgba(255, 255, 255, 0.7)",
      },
      "menu": Object {
        "backgroundColor": "#303030",
        "containerBackgroundColor": "#303030",
      },
      "menuItem": Object {
        "dataHeight": 32,
        "height": 48,
        "hoverColor": "rgba(255, 255, 255, 0.1)",
        "padding": 24,
        "rightIconDesktopFill": "#757575",
        "selectedTextColor": "#ff4081",
      },
      "menuSubheader": Object {
        "borderColor": "rgba(255, 255, 255, 0.3)",
        "padding": 24,
        "textColor": "#0097a7",
      },
      "overlay": Object {
        "backgroundColor": "rgba(0, 0, 0, 0.54)",
      },
      "palette": Object {
        "accent1Color": "#ff4081",
        "accent2Color": "#f50057",
        "accent3Color": "#ff80ab",
        "alternateTextColor": "#303030",
        "borderColor": "rgba(255, 255, 255, 0.3)",
        "canvasColor": "#303030",
        "clockCircleColor": "rgba(255, 255, 255, 0.12)",
        "disabledColor": "rgba(255, 255, 255, 0.3)",
        "pickerHeaderColor": "rgba(255, 255, 255, 0.12)",
        "primary1Color": "#0097a7",
        "primary2Color": "#0097a7",
        "primary3Color": "#757575",
        "secondaryTextColor": "rgba(255, 255, 255, 0.7)",
        "shadowColor": "rgba(0, 0, 0, 1)",
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "paper": Object {
        "backgroundColor": "#303030",
        "color": "rgba(255, 255, 255, 1)",
        "zDepthShadows": Array [
          "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
          "0 3px 10px rgba(0, 0, 0, 0.16),
         0 3px 10px rgba(0, 0, 0, 0.23)",
          "0 10px 30px rgba(0, 0, 0, 0.19),
         0 6px 10px rgba(0, 0, 0, 0.23)",
          "0 14px 45px rgba(0, 0, 0, 0.25),
         0 10px 18px rgba(0, 0, 0, 0.22)",
          "0 19px 60px rgba(0, 0, 0, 0.3),
         0 15px 20px rgba(0, 0, 0, 0.22)",
        ],
      },
      "prepareStyles": [Function],
      "radioButton": Object {
        "backgroundColor": "#303030",
        "borderColor": "rgba(255, 255, 255, 1)",
        "checkedColor": "#0097a7",
        "disabledColor": "rgba(255, 255, 255, 0.3)",
        "labelColor": "rgba(255, 255, 255, 1)",
        "labelDisabledColor": "rgba(255, 255, 255, 0.3)",
        "requiredColor": "#0097a7",
        "size": 24,
      },
      "raisedButton": Object {
        "color": "#303030",
        "disabledColor": "rgb(43, 43, 43)",
        "disabledTextColor": "rgba(255, 255, 255, 0.3)",
        "fontSize": 14,
        "fontWeight": 500,
        "primaryColor": "#0097a7",
        "primaryTextColor": "#303030",
        "secondaryColor": "#ff4081",
        "secondaryTextColor": "#303030",
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "rawTheme": Object {
        "fontFamily": "Roboto, sans-serif",
        "palette": Object {
          "accent1Color": "#ff4081",
          "accent2Color": "#f50057",
          "accent3Color": "#ff80ab",
          "alternateTextColor": "#303030",
          "borderColor": "rgba(255, 255, 255, 0.3)",
          "canvasColor": "#303030",
          "clockCircleColor": "rgba(255, 255, 255, 0.12)",
          "disabledColor": "rgba(255, 255, 255, 0.3)",
          "pickerHeaderColor": "rgba(255, 255, 255, 0.12)",
          "primary1Color": "#0097a7",
          "primary2Color": "#0097a7",
          "primary3Color": "#757575",
          "secondaryTextColor": "rgba(255, 255, 255, 0.7)",
          "shadowColor": "rgba(0, 0, 0, 1)",
          "textColor": "rgba(255, 255, 255, 1)",
        },
        "spacing": Object {
          "desktopDrawerMenuItemHeight": 48,
          "desktopDropDownMenuFontSize": 15,
          "desktopDropDownMenuItemHeight": 32,
          "desktopGutter": 24,
          "desktopGutterLess": 16,
          "desktopGutterMini": 8,
          "desktopGutterMore": 32,
          "desktopKeylineIncrement": 64,
          "desktopSubheaderHeight": 48,
          "desktopToolbarHeight": 56,
          "iconSize": 24,
        },
      },
      "refreshIndicator": Object {
        "loadingStrokeColor": "#0097a7",
        "strokeColor": "rgba(255, 255, 255, 0.3)",
      },
      "ripple": Object {
        "color": "rgba(255, 255, 255, 0.87)",
      },
      "slider": Object {
        "handleColorZero": "#757575",
        "handleFillColor": "#303030",
        "handleSize": 12,
        "handleSizeActive": 18,
        "handleSizeDisabled": 8,
        "rippleColor": "#0097a7",
        "selectionColor": "#0097a7",
        "trackColor": "#757575",
        "trackColorSelected": "#ff80ab",
        "trackSize": 2,
      },
      "snackbar": Object {
        "actionColor": "#ff4081",
        "backgroundColor": "rgba(255, 255, 255, 1)",
        "textColor": "#303030",
      },
      "spacing": Object {
        "desktopDrawerMenuItemHeight": 48,
        "desktopDropDownMenuFontSize": 15,
        "desktopDropDownMenuItemHeight": 32,
        "desktopGutter": 24,
        "desktopGutterLess": 16,
        "desktopGutterMini": 8,
        "desktopGutterMore": 32,
        "desktopKeylineIncrement": 64,
        "desktopSubheaderHeight": 48,
        "desktopToolbarHeight": 56,
        "iconSize": 24,
      },
      "stepper": Object {
        "backgroundColor": "transparent",
        "connectorLineColor": "#bdbdbd",
        "disabledTextColor": "rgba(0, 0, 0, 0.26)",
        "hoverBackgroundColor": "rgba(0, 0, 0, 0.06)",
        "hoveredIconColor": "#616161",
        "iconColor": "#0097a7",
        "inactiveIconColor": "#9e9e9e",
        "textColor": "rgba(0, 0, 0, 0.87)",
      },
      "subheader": Object {
        "color": "rgba(255, 255, 255, 0.54)",
        "fontWeight": 500,
      },
      "svgIcon": Object {
        "color": "rgba(255, 255, 255, 1)",
      },
      "table": Object {
        "backgroundColor": "#303030",
      },
      "tableFooter": Object {
        "borderColor": "rgba(255, 255, 255, 0.3)",
        "textColor": "#ff80ab",
      },
      "tableHeader": Object {
        "borderColor": "rgba(255, 255, 255, 0.3)",
      },
      "tableHeaderColumn": Object {
        "height": 56,
        "spacing": 24,
        "textColor": "#ff80ab",
      },
      "tableRow": Object {
        "borderColor": "rgba(255, 255, 255, 0.3)",
        "height": 48,
        "hoverColor": "#f50057",
        "selectedColor": "rgba(255, 255, 255, 0.3)",
        "stripeColor": "rgba(127, 203, 211, 0.4)",
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "tableRowColumn": Object {
        "height": 48,
        "spacing": 24,
      },
      "tabs": Object {
        "backgroundColor": "#0097a7",
        "selectedTextColor": "#303030",
        "textColor": "rgba(48, 48, 48, 0.7)",
      },
      "textField": Object {
        "backgroundColor": "transparent",
        "borderColor": "rgba(255, 255, 255, 0.3)",
        "disabledTextColor": "rgba(255, 255, 255, 0.3)",
        "errorColor": "#f44336",
        "floatingLabelColor": "rgba(255, 255, 255, 0.3)",
        "focusColor": "#0097a7",
        "hintColor": "rgba(255, 255, 255, 0.3)",
        "textColor": "rgba(255, 255, 255, 1)",
      },
      "timePicker": Object {
        "accentColor": "#0097a7",
        "clockCircleColor": "rgba(255, 255, 255, 0.12)",
        "clockColor": "rgba(255, 255, 255, 1)",
        "color": "#303030",
        "headerColor": "rgba(255, 255, 255, 0.12)",
        "selectColor": "#0097a7",
        "selectTextColor": "#303030",
        "textColor": "#303030",
      },
      "toggle": Object {
        "labelColor": "rgba(255, 255, 255, 1)",
        "labelDisabledColor": "rgba(255, 255, 255, 0.3)",
        "thumbDisabledColor": "rgba(255, 255, 255, 0.3)",
        "thumbOffColor": "#f50057",
        "thumbOnColor": "#0097a7",
        "thumbRequiredColor": "#0097a7",
        "trackDisabledColor": "#757575",
        "trackOffColor": "#757575",
        "trackOnColor": "rgba(0, 151, 167, 0.5)",
        "trackRequiredColor": "rgba(0, 151, 167, 0.5)",
      },
      "toolbar": Object {
        "backgroundColor": "rgb(232, 0, 82)",
        "color": "rgba(255, 255, 255, 0.54)",
        "height": 56,
        "hoverColor": "rgba(255, 255, 255, 0.87)",
        "iconColor": "rgba(255, 255, 255, 0.4)",
        "menuHoverColor": "rgba(255, 255, 255, 0.1)",
        "separatorColor": "rgba(255, 255, 255, 0.175)",
        "titleFontSize": 20,
      },
      "tooltip": Object {
        "color": "#ffffff",
        "opacity": 0.9,
        "rippleBackgroundColor": "#616161",
      },
      "userAgent": undefined,
      "zIndex": Object {
        "appBar": 1100,
        "dialog": 1500,
        "dialogOverlay": 1400,
        "drawer": 1300,
        "drawerOverlay": 1200,
        "layer": 2000,
        "menu": 1000,
        "popover": 2100,
        "snackbar": 2900,
        "tooltip": 3000,
      },
    }
  }
>
  <div>
    <Navbar />
    <Search />
  </div>
</MuiThemeProvider>
`;
